packages/react/src/utilities/positioning/positioning.ts

```diff
@@ -212,6 +212,20 @@ function _getOutOfBoundsDegree(rect: Rectangle, bounds: Rectangle) {
   return total;
 }
 
+/**
+ * Returns true if scroll-resizing will move the target edge within the bounding rectangle,
+ * and there is room between the target edge and the bounding edge for scrolled content.
+ * Returns false otherwise.
+ */
+function _canScrollResizeToFitEdge(target: Rectangle, bounding: Rectangle, targetEdge: RectangleEdge) {
+  // Only scroll vertically to fit - cannot scroll to fit right or left edges
+  if (targetEdge !== RectangleEdge.bottom && targetEdge !== RectangleEdge.top) {
+    return false;
+  }
+
+  return _getRelativeEdgeDifference(target, bounding, targetEdge) > 100;
```

Comment:
I chose 100 arbitrarily here as the amount of space that we can reasonably scroll-resize down to (and I don't think it's the right value - this is too small of a space). Would it make sense to allow consumers to define this in props, and have a default value?

Reply:
I ended up adding a `minimumScrollResizeHeight` prop to Callout